osu.Game/Overlays/News/NewsCard.cs

```diff
@@ -31,24 +31,28 @@ public class NewsCard : CompositeDrawable
         public NewsCard(APINewsPost post)
         {
             this.post = post;
-        }
 
-        [BackgroundDependencyLoader]
-        private void load()
-        {
             RelativeSizeAxes = Axes.X;
             AutoSizeAxes = Axes.Y;
             Masking = true;
             CornerRadius = 6;
+        }
 
-            NewsBackground bg;
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider, GameHost host)
+        {
+            if (post.Slug != null)
+            {
+                TooltipText = "view in browser";
+                Action = () => host.OpenUrlExternally("https://osu.ppy.sh/home/news/" + post.Slug);
+            }
```

Comment:
this has weird UX with the post date; if you hover over the date, the date details tooltip will replace the `view in browser` one, but clicking will still open the post in the browser. not sure what to do with that; I guess I'd be fine with the date suppressing clicks if it's deemed a problem.

Reply:
Now date suppressing clicks